src/components/stream_screen/stream_screen.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import MinimizedPost from "../MinimizedPost/index";
+
+export default function StreamScreen(props) {
+  var posts = [];
+  for (let i = 0; i < props.postArr.length; i++) {
+    posts.push(
+      <MinimizedPost
+        key={props.postArr[i].id}
+        postTitle={props.postArr[i].title}
+        postAuthor={props.postArr[i].author}
+      />
+    );
+  }
+  return <div>{posts}</div>;
```

Comment:
Use [Array.map method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to get the array of `MinimizedPost` components, instead of using a for loop. You can write the whole component then similar to:
```
export default function StreamScreen(props) {
  return (<div>
    props.postArr.map(post => <MinimizedPost ...props... />)
  </div>);
}
```